src/theory/ff/theory_ff.cpp

```diff
@@ -77,70 +80,106 @@ void TheoryFiniteFields::finishInit()
 
 void TheoryFiniteFields::postCheck(Effort level)
 {
-  // TODO
+#ifdef CVC5_USE_COCOA
+  Trace("ff::check") << "ff::check : " << level << std::endl;
+  for (auto& subTheory : d_subTheories)
+  {
+    subTheory.second.postCheck(level);
+    if (subTheory.second.inConflict())
+    {
+      d_im.conflict(
+          NodeManager::currentNM()->mkAnd(subTheory.second.conflict()),
+          InferenceId::FF_LEMMA);
+    }
+  }
+#else  /* CVC5_USE_COCOA */
+  // We don't crash here, since we'd have crashed on notifyFact
```

Comment:
Can this be checked with an assertion? Maybe `Unreachable();`?

Reply:
Yeah, good idea. I've changed this to crash if we've seen any field terms.

This code is reached if we're in QF_ALL, even if there are no field terms.

Reply:
Okay, actually, that change didn't work.

My idea was to assert that we're received no facts, but in the `--no-cocoa` configuration, we don't actually have any data structure that records the facts: we just crash if we ever get any.

I'm not sure if there is a good assertion to add here...